docs/consume-packages/Package-Restore.md

```diff
@@ -115,6 +115,15 @@ To enable or disable Package Restore for all users on a computer, a developer or
 > [!Important]
 > If you edit the `packageRestore` settings directly in `nuget.config`, restart Visual Studio, so that the **Options** dialog box shows the current values.
 
+### Choose default package management format
+
+![Control default package management format though NuGet Package Manager options](media/Restore-02-PackageFormatOptions.png)
+
+NuGet has two formats in which a project may use packages, `PackageReference` and `packages.config`. The default format can be selected from the drop down under the **Package Management** heading. An option to be prompted when the first package is installed in a project is also available.
```

Comment:
I think you can use a colon, like "may use packages:". Also dropdown or drop-down seems better.